### PR TITLE
Instructor feedback

### PR DESCRIPTION
**Pull request by VARSHAV2001**

### CODE REVIEW

File: client/src/assets/style.css [hunk 1]
```diff
@@ -64,4 +64,19 @@
     #search-box input {
         width: 15rem;
     }
-}
\ No newline at end of file
+}
+
+.feedback-header{
+    background-color: white;
```
Comment (mishelshaji) [inline client/src/assets/style.css]: Use color value instead.

File: client/src/assets/style.css [hunk 2]
```diff
@@ -64,4 +64,19 @@
...
+ }
+ 
+ .feedback-body-content{
+     background-color: white;
```
Comment (mishelshaji) [inline client/src/assets/style.css]: Use color value instead.

File: client/src/assets/style.css [hunk 3]
```diff
@@ -64,4 +64,19 @@
...
+ }
+ 
+ .feedback-main-body{
```
Comment (mishelshaji) [inline client/src/assets/style.css]: Format the code.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/src/assets/style.css (hunk 1 above)
Comment (VARSHAV2001) [reply client/src/assets/style.css]: Issue resolved

File: client/src/assets/style.css (hunk 2 above)
Comment (VARSHAV2001) [reply client/src/assets/style.css]: Issue resolved

File: client/src/assets/style.css (hunk 3 above)
Comment (VARSHAV2001) [reply client/src/assets/style.css]: Issue resolved

**Review by mishelshaji — APPROVED** [review]

File: client/src/App.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ import StudentRegistration from './pages/student/StudentRegistration';
 import InstructorRegistration from './pages/instructor/InstructorRegistration';
 import EnrolledCourses from './pages/student/EnrolledCourses';
 import Order from './pages/student/Order';
+import FeedbackList from './pages/instructor/FeedbackList';
 import StudentProfile from './pages/student/StudentProfile';
```
Comment (mishelshaji) [inline client/src/App.js]: Format and group routes.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 4 above)
Comment (VARSHAV2001) [reply client/src/App.js]: formatted

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**